benefits/core/migrations/local_fixtures.json

```diff
@@ -185,101 +145,36 @@
-      "unverified_template": "eligibility/unverified--mst-courtesy-card.html",
-      "help_template": "core/includes/help--mst-courtesy-card.html"
+      "selection_label_template": "eligibility/includes/selection-label--cst-agency-card.html",
+      "start_template": "eligibility/start--cst-agency-card.html",
+      "form_class": "benefits.eligibility.forms.CSTAgencyCard",
+      "unverified_template": "eligibility/unverified--cst-agency-card.html",
+      "help_template": "core/includes/help--cst-agency-card.html"
     }
   },
   {
     "model": "core.eligibilityverifier",
     "pk": 4,
     "fields": {
-      "name": "(SacRT) oauth claims via Login.gov",
-      "display_order": 5,
-      "active": false,
-      "api_url": null,
-      "api_auth_header": null,
-      "api_auth_key_secret_name": null,
-      "eligibility_type": 4,
-      "public_key": null,
-      "jwe_cek_enc": null,
-      "jwe_encryption_alg": null,
-      "jws_signing_alg": null,
-      "auth_provider": 1,
-      "selection_label_template": "eligibility/includes/selection-label--senior.html",
-      "start_template": "eligibility/start--senior.html",
-      "form_class": null
-    }
-  },
-  {
-    "model": "core.eligibilityverifier",
-    "pk": 5,
-    "fields": {
-      "name": "(SBMTD) oauth claims via Login.gov",
-      "display_order": 6,
-      "active": false,
-      "api_url": null,
-      "api_auth_header": null,
-      "api_auth_key_secret_name": null,
-      "eligibility_type": 5,
-      "public_key": null,
-      "jwe_cek_enc": null,
-      "jwe_encryption_alg": null,
-      "jws_signing_alg": null,
-      "auth_provider": 1,
-      "selection_label_template": "eligibility/includes/selection-label--senior.html",
-      "start_template": "eligibility/start--senior.html",
-      "form_class": null
-    }
-  },
-  {
-    "model": "core.eligibilityverifier",
-    "pk": 6,
-    "fields": {
-      "name": "(SBMTD) eligibility server verifier",
-      "display_order": 7,
-      "active": true,
-      "api_url": "http://server:8000/verify",
-      "api_auth_header": "X-Server-API-Key",
-      "api_auth_key_secret_name": "mobility-pass-verifier-api-auth-key",
-      "eligibility_type": 6,
-      "public_key": 2,
-      "jwe_cek_enc": "A256CBC-HS512",
-      "jwe_encryption_alg": "RSA-OAEP",
-      "jws_signing_alg": "RS256",
-      "auth_provider": null,
-      "selection_label_template": "eligibility/includes/selection-label--sbmtd-mobility-pass.html",
-      "start_template": "eligibility/start--sbmtd-mobility-pass.html",
-      "form_class": "benefits.eligibility.forms.SBMTDMobilityPass",
-      "unverified_template": "eligibility/unverified--sbmtd-mobility-pass.html",
-      "help_template": "core/includes/help--sbmtd-mobility-pass.html"
-    }
-  },
-  {
-    "model": "core.eligibilityverifier",
-    "pk": 7,
-    "fields": {
-      "name": "(MST) CalFresh oauth claims via Login.gov",
+      "name": "(CST) CalFresh oauth claims via Login.gov",
```

Comment:
(Not for this PR): eventually we should have a new `core.authprovider` instance for this verifier:

```json
{
  "model": "core.authprovider",
  "pk": 3,
  "fields": {
    "sign_out_button_template": "core/includes/button--sign-out--login-gov.html",
    "sign_out_link_template": "core/includes/link--sign-out--login-gov.html",
    "client_name": "calfresh-benefits-oauth-client-name",
    "client_id_secret_name": "auth-provider-client-id",
    "authority": "https://example.com",
    "scope": "verify:calfresh",
    "claim": "calfresh",
    "scheme": "dev-cal-itp_benefits"
  }
},
```

Then update the field here on the `eligibilityverifier`:

```diff
- "auth_provider": 1
+ "auth_provider": 3
```